lms/services/assignment.py

```diff
@@ -211,18 +209,6 @@ def is_member(self, assignment: Assignment, h_userid: str) -> bool:
             assignment.membership.join(User).filter(User.h_userid == h_userid).first()
         )
 
-    def get_members(
```

Comment:
Removing this method.

Instead of getting the members of one assignment we switch to get users and filter by the assignment_id.

This change should be flexible longer term as we can filter by course, assignment, user in the same endpoint.